fastai/data_block.py

```diff
@@ -419,6 +427,18 @@ def from_lists(cls, path:PathOrStr, inputs, labels)->'LabelList':
     def transform(self, tfms:TfmList, tfm_y:bool=None, **kwargs):
         "Set the `tfms` and `` tfm_y` value to be applied to the inputs and targets."
         self.tfms,self.tfmargs = tfms,kwargs
-        if tfm_y is not None: self.tfm_y=tfm_y
+        if tfm_y is not None:
+            self.tfm_y=tfm_y
+            self.tfms_y=tfms
```

Comment:
Don't we need to pass the kwargs to self.tfmargs_y here? I think this is going to make camvid and object detect fail.

Reply:
yes you're right, will fix that